xbmc/windowing/gbm/DRMUtils.cpp

```diff
@@ -737,3 +741,23 @@ uint32_t CDRMUtils::FourCCWithoutAlpha(uint32_t fourcc)
 {
   return (fourcc & 0xFFFFFF00) | static_cast<uint32_t>('X');
 }
+
+bool CDRMUtils::CheckConnector(int connector_id)
```

Comment:
could you possibly just pass through the entire `drmModeConnectorPtr` to avoid the extra boilerplate?

Reply:
I'm not sure what you mean.
I have to call drmModeGetConnector again anyway in order to see the change on the connector.
I don't want to pass the original pointer and overwrite it, I just want to save the current state on the local variable, I think changing the real connector member would cause issues, wouldn't it?

Reply:
ah yes, makes sense